applications/search/src/pages/dataset-details-page/dataset-details-page-pure.jsx

```diff
@@ -186,23 +187,35 @@ const renderContactPoints = dataset => {
   );
 };
 
+const resolveLinkOrExternalLink = item => {
+  if (!getConfig().themeNap) {
+    return (
+      <Link
+        to={`/concepts/${item.uri.substring(item.uri.lastIndexOf('/') + 1)}`}
```

Comment:
url er samme til begge nap og fdk, så du kan regne det ut i en var

Reply:
Rettet. Men for <Link> kan jeg bare bruke relativ path.